app/scripts/vueCore.js

```diff
@@ -48,7 +48,11 @@
               // TODO: need to update setting timeout
               var interval = setTimeout(function() {
                 clearInterval(interval);
-                self_.$broadcast('update-special-charts');
+                var _attrs = [];
+                _.each(self_.groups, function(group) {
+                  _attrs = _attrs.concat(group.attributes);
+                });
+                self_.$broadcast('update-special-charts', _attrs);
```

Comment:
I think hasfilter variable would be sufficient instead of sending all group attributes